src/components/Header/Header.module.scss

```diff
@@ -29,13 +34,39 @@
     }
 }
 
-// Hide | for the last elements
 .header a:last-child {
     .link::after {
         display: none;
     }
 }
+// Remove upto this
```

Comment:
Comment if not needed, please remove

Reply:
I have added this comment for future reference, when this current PR is out of feature flag, these classes should be removed. I added this since removing css classes are forgotten many times.

Reply:
Got it
But "Remove upto this" does not convey much 
When, Why ?

Reply:
The starting line comment is on line 25. Should I change the end line comment?